pkg/arvo/sys/vane/ames.hoon

```diff
@@ -2004,86 +2096,82 @@
     |=  =ship
     ^+  event-core
     =+  (trace msg.veb ship |.("requesting attestion"))
-    =.  event-core  (send-blob | ship (sendkeys-packet ship))
+    =.  event-core  (send-packet | ~ (sendkeys-packet ship))
     =/  =wire  /alien/(scot %p ship)
     (emit duct %pass wire %b %wait (add now ~s30))
-  ::  +send-blob: fire packet at .ship and maybe sponsors
-  ::
-  ::    Send to .ship and sponsors until we find a direct lane,
-  ::    skipping .our in the sponsorship chain.
+  ::  +send-packet: fire packet at lane and maybe next
   ::
   ::    If we have no PKI data for a recipient, enqueue the packet and
   ::    request the information from Jael if we haven't already.
   ::
-  ++  send-blob
-    ~/  %send-blob
-    |=  [for=? =ship =blob]
+  ++  send-packet
+    |=  [for=? next-lane=(unit lane) =packet]
```

Comment:
As next-lane is a unit, is `for` redundant?

Reply:
yes I think so I'll get rid of it